host_vars/dalite.harvardx.harvard.edu/public.yml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
Abstract: Not really, we can specify global overrides, and we can't override variables for a given host. 

Not really. The problem is the variable file precedence in Ansible. Ansible people also dislike overriding variables, and say that: "If you have to figure out variable from which file is used you are doing something wrong" (I'm not saying that this right, but that's the general attitude). 

Anyways, ``host_vars`` and ``group_vars`` can be in two places: in the directory where the executed playbook is (in our case it's deploy repository --- or this repository) and in the directory where ansible.cfg (in our case secrets repository). However for ansible ``host_vars`` from ```deploy`` repository takes precedence. 

The approach I use is to define in public.yml things like: 

``TARSNAP_KEY = "{{ MYSQL_TARSNAP_KEY }}"``

and then in private yml I can set tarsnap key, for only a single instance setting ``MYSQL_TARSNAP_KEY``.